R/compute_consensus.R

```diff
@@ -123,7 +136,8 @@ compute_consensus <- function(model_fit, type = "CP", burnin = model_fit$burnin,
   # Find the CP consensus per cluster, using the find_cpc function
   df <- dplyr::ungroup(df)
   df <- dplyr::group_by(df, .data$cluster)
-  df <- dplyr::do(df, find_cpc(.data))
+  class(df) <- c("BayesMallows", "grouped_df", "tbl_df", "tbl", "data.frame")
+  df <- find_cpc(df)
```

Comment:
Numerically, it seems like the results are correct. However, since `compute_consensus` is a user-facing function, I think it is a good thing to have the results sorted. I'll take a look and get back to you.

Reply:
I'm not sure, because I haven't thought about it before. Taking a look at the code in this branch now.